packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -140,22 +139,15 @@ function OnboardingModal({
   const content = (() => {
     if (isAuthenticating) {
       return (
-        <>
-          <CloseButton
-            className="top-2 right-2 z-2"
-            style={{ position: 'absolute' }}
-            onClick={onClose}
-          />
-          <AuthOptions
-            className="h-full"
-            onClose={onClose}
-            formRef={formRef}
-            onSuccessfulLogin={onRegistrationSuccess}
-            onSuccessfulRegistration={onRegistrationSuccess}
-            trigger={AuthTriggers.CreateFeedFilters}
-            onDisplayChange={(display: AuthDisplay) => setScreenValue(display)}
-          />
-        </>
+        <AuthOptions
```

Comment:
Why did we remove the CloseButton here?

Reply:
We already render CloseButton in the `AuthHeader` component. This is rendered for each tab in `AuthOptions`. So we were effectively rendering 2 close buttons on top of each other. I decided to remove this one to only have 1 button there.

Reply:
Got it, thanks for the info :)